pkgs/misc/vscode-extensions/update_installed_exts.sh

```diff
@@ -63,12 +122,22 @@ trap clean_up SIGINT
 printf '{ extensions = [\n'
 
 # Note that we are only looking to update extensions that are already installed.
-for i in $($CODE --list-extensions)
-do
-    OWNER=$(echo "$i" | cut -d. -f1)
-    EXT=$(echo "$i" | cut -d. -f2)
+(
+    set +e
```

Comment:
is there something in here which is likely to fail? Maybe we should just append a ``|| true`` to that command instead.

Reply:
The script is meant to run even if some extensions are not updated successfully.
Suggestions applied.